Amazon.Ion.ObjectMapper.Test/Utils.cs

```diff
@@ -70,7 +70,7 @@ public static void Check<T>(IonSerializer ionSerializer, T item)
 
         public static void AssertHasAnnotation(string annotation, Stream stream)
         {
-            AssertHasAnnotation(annotation, StreamToIonValue(Copy(stream)));
+            AssertHasAnnotation(annotation, StreamToIonValue(stream));
```

Comment:
`Copy()` is now happening inside `StreamToIonValue()`

Reply:
@justing-bq I don't think that answers the question - is the Copy needed?

Reply:
The copy was removed from this line because I moved the copy to inside the function.
The reason for the copy is so we can continue reusing the stream without worrying about resetting the position.